docusaurus.config.js

```diff
@@ -185,8 +185,8 @@ module.exports = {
         {
           items: [
             {
-              label: "Join the Cloud Waitlist",
-              href: "https://us17.list-manage.com/survey?u=2334a0f23e55fd1840613755d&id=f1895b6f4a",
+              label: "Join the Cloud waitlist",
```

Comment:
```suggestion
              label: "Join the Cloud Waitlist",
```
Oops. Would've been smart for me to check the latest changes to the footer first.